doc/run-models.md

```diff
@@ -0,0 +1,107 @@
+# How to produce population projections
+
+## Introduction
+`witan.models.demography` is a Clojure library to run demographic models.
+
+The aim of `witan.models.demography` is to offer "generic" models in the sense that our models will be adaptable to different methodologies and data inputs. To learn more about the methodology, see the [modelling docs](intro.md). To learn more about the data inputs currently being used, jump to the [data inputs](#data-inputs) section.
+
+A minimal version of the population projection is now available for English unitary authorities, non-metropolitan districts, and boroughs. Projections are also available for the countries of Northern Ireland, Scotland, and Wales. 
+
+## Running a population projection
+
+An executable.jar file, which contains all the code and data needed to produce a projection, can be downloaded from http://example.com/FIXME.
+
+The .jar file contains the following:
+
+![minimal CCM](images/jar-setup.png)
+
+To run the .jar file and produce a projection, follow the instructions in the [README](repo/blob/master/README.md).
+
+## User-defined parameters for the model
+
+Some parameters in the model need to be specified by the user. The .jar file contains default values for the parameters, which can be changed as needed. The parameters, with defaults in parentheses, are:
```

Comment:
Saying the .jar file contains the default values for the parameters isn't really true as they're actually held in the config file, but saying the jar will access the defaults if no changes are made may simplify the understanding for some users